code/game/mecha/mecha.dm

```diff
@@ -817,7 +828,7 @@
 	if(istype(W, /obj/item/mecha_parts/mecha_equipment))
 		var/obj/item/mecha_parts/mecha_equipment/E = W
 		spawn()
-			if(E.can_attach(src))
+			if((E.can_attach(src) || is_killdozer()) && get_remaining_equipment_slots())
```

Comment:
will certainly cause runtimes if you attach a clamp to a combat mech

Reply:
not seeing runtimes but yeah the clamp doesn't work on them

Reply:
lacks cargo capacity